src/pages/Welcome/Welcome.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../styles/Colors.scss";
```

Comment:
add imports to globals.scss because they're included to app (styles are global in scss)

Reply:
You need webpack for that.